Packs/ApiModules/Scripts/CoreXQLApiModule/CoreXQLApiModule.py

```diff
@@ -641,6 +647,18 @@ def start_xql_query_polling_command(client: CoreClient, args: dict) -> Union[Com
     if not args.get('query_name'):
         raise DemistoException('Please provide a query name')
     execution_id = start_xql_query(client, args)
+    if execution_id == 'FAILURE':
+        # the 'start_xql_query' function failed because it reached the maximum allowed number of parallel running queries.
+        # running the command again using polling with an interval of 'interval_in_secs' seconds.
+        command_results = CommandResults()
+        interval_in_secs = int(args.get('interval_in_seconds', 10))
```

Comment:
is this arg already exist for this command? Also, do we really need 10 second intervals by default? maybe less...

Reply:
exist.
ill check if shorter intervals will work the same as 10 seconds